src/__tests__/CompareResults/beta/RevisionSelect.test.tsx

```diff
@@ -86,12 +95,23 @@ describe('Revision select', () => {
 
     fireEvent.mouseDown(selectButton);
 
-    // check every revision header to be for revision bb6a5e451dac
-    comparisonHeaders = await screen.findAllByTestId(/revision-header/);
-
-    comparisonHeaders.forEach((resultHeader) =>
-      expect(resultHeader.textContent).toContain('bb6a5e451dac'),
-    );
+    // check to display results only for revision bb6a5e451dac
+    firstRevisionHeaders = await screen.findAllByText(/bb6a5e451dac/, {
```

Comment:
Maybe add a comment about the fact that findByRole doesn't work anymore because the root element has `aria-hidden` but we don't know why yet?